src/components/Form/EntryField.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Controller } from 'react-hook-form'
+
+// FPCC
+import getIcon from 'common/utils/getIcon'
+import Modal from 'components/Modal'
+import EntrySelector from 'components/EntrySelector'
+import { TYPE_WORD, TYPE_PHRASE } from 'common/constants'
+import ValidationError from 'components/Form/ValidationError'
+
+function EntryField({
```

Comment:
EntryArrayField takes maxItems-- is there a reason we can't use that with maxItems set to 1? Maybe we could pop the value out of the array in a data adaptor.

Reply:
Good call. There was a reason why I didn't do this but I'm having trouble remembering what it was lol. I'll test it out and see if I can make it work. 👍